Objects/memoryobject.c

```diff
@@ -1689,6 +1690,8 @@ unpack_single(const char *ptr, const char *fmt)
     unsigned char uc;
     void *p;
 
+    CHECK_RELEASED(self); /* See gh-92888 for why we need this here */
```

Comment:
Rather than copy/paste the same comment, maybe you can add a new macro with a more explicit name, and put a long comment explaining why this macro is needed. The implementation would just call CHECK_RELEASED().

For example, CHECK_AGAIN_RELEASED().